modules/person.rb

```diff
@@ -0,0 +1,45 @@
+# top level comment for linter error
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def to_get_id
+    id
+  end
+
+  def to_get_name
+    @name
+  end
+
+  def to_get_age
+    @age
+  end
+
+  def to_set_name=(name)
+    @name = name
+  end
+
+  def to_set_age=(age)
+    @age = age
+  end
+
```

Comment:
- [ ] Do we really need al these methods just to get and set attributes? We already have Ruby getter and setter: 

  - `attr_reader` = getter
  - `attr_accessor` = getter and setter

  Kindly remove these methods and use the above suggestion to set reader for `:id` and getter-setter for `:name` and `:age`